### ISSUE TITLE
Submitted fonts

### ISSUE THREAD

**Issue by shahabsiavash** [google/fonts#286]
Hi,

I sent 2 Persian fonts about a week ago and another 2 tonight. I didn't receive any replies. I hope we can see more **Persian (Arabic) typefaces** specially display ones in Google Fonts. 

I saw the **Lalezar** font by Borna Izadpanah and it would be nice if we see more there.

**Comment by davelab6** [comment]
@shahabsiavash thanks for reaching out here too. Please see https://github.com/google/fonts/blob/master/CONTRIBUTING.md to learn more about the requirements :) Feel free to update this issue with more details about the 4 fonts you have in mind if they meet the requirements :)

**Comment by shahabsiavash** [comment]
Thanks for the reply. I sent the files via Google Fonts Form. Now, I guess it is not the right way to do that! :)

And also my Persian fonts are only support Arabic and Persian. Are Latin glyphs mandatory? 

Great day ;)

**Comment by davelab6** [comment]
Yep, all font files must have at a minimum the 215 glyphs listed in the `latin_unique-glyphs.nam` file

**Comment by alexeiva** [comment]
@shahabsiavash if you don't want to draw Latin from scratch, you can borrow them from another OFL font

**Comment by shahabsiavash** [comment]
@alexeiva Thanks a lot. Is there any page or group that I can find some of these? Because I want to be 100% sure that they are OFL and also can get in touch with the designer.

**Comment by alexeiva** [comment]
@shahabsiavash Sure! All fonts on fonts.google.com are opensource. Most of which are OFL, while others may be under Apache or UFL.

You can browse all OFL's here:
https://github.com/google/fonts/tree/master/ofl

You are welcome to post your design-in-progress or any related questions on the GF Discussions forum:
https://groups.google.com/forum/#!forum/googlefonts-discuss

**Comment by shahabsiavash** [comment]
@alexeiva Thanks. I joined. the group :)

**Comment by shahabsiavash** [comment]
Hi, I designed the Latin glyphs and it is almost finished.

I posted it on [Google fonts discussion group](https://groups.google.com/forum/#!forum/googlefonts-discuss).  

Here's the link:
[Review Request: New Persian (Arabic) typeface](https://groups.google.com/d/msg/googlefonts-discuss/3bK0A59NOkc/TZVVzy1jCwAJ)

Should I update this issue after it is finished or open a new one?

Thanks,
@davelab6 @alexeiva

**Comment by davelab6** [comment]
Great :) updating here is good

**Comment by shahabsiavash** [comment]
I asked because the `new family` tag has been removed. :)
@davelab6

**Comment by davelab6** [comment]
I moved it to the new family milestone :)

**Comment by shahabsiavash** [comment]
Hi,

Finally here! I designed the Latin glyphs, completed the Persian and did the metrics, kerning, {salt} feature and I also added 9 more styles to the family including a Pure Persian style.

This font family includes the first ever [Persian color font](https://github.com/shahabsiavash/KayKhosrow-Font/tree/master/dist/KayKhosrow-Colored) :) [Layers, SVG, sbix, COLR] 

Please read the [ReadMe](https://github.com/shahabsiavash/KayKhosrow-Font/blob/master/README.md) for more information.

Also got some good feedbacks and insights from TypeDrawers masters in the mean time.

**Here is the project:
[KayKhosrow-Font](https://github.com/shahabsiavash/KayKhosrow-Font)**


Please let me know what are the next steps. I'm excited about this :)

Thanks and have a great day,

@davelab6 @alexeiva

**Comment by davelab6** [comment]
@shahabsiavash alright! 

```
KayKhosrow-Light.ttf
KayKhosrow-Bold.ttf
KayKhosrow-Black.ttf
```

Are these 3 weights the same design on a weight continuum?

If so, I will need a `KayKhosrow-Regular.ttf` :)

```
KayKhosrow-Italic.ttf
```

Ideally I'd like an italic for each roman weight, in the above list.   

```
KayKhosrow-Oblique.ttf
KayKhosrow-Outline.ttf
KayKhosrow-PurePersian.ttf
KayKhosrow-Round.ttf	
KayKhosrow-Smooth.ttf
KayKhosrow-Inline.ttf
```

Each of these should be its own family, in a Regular weight. Eg,

```
KayKhosrowOblique-Regular.ttf
KayKhosrowOutline-Regular.ttf
KayKhosrowPurePersian-Regular.ttf
KayKhosrowRound-Regular.ttf	
KayKhosrowSmooth-Regular.ttf
KayKhosrowInline-Regular.ttf
```

And this is also true for `KayKhosrow-Colored`, but currently Google Fonts isn't supporting color fonts.

**Comment by shahabsiavash** [comment]
`If so, I will need a KayKhosrow-Regular.ttf :)`

Actually it was Regular at first, but I changed it to light. I'm changing it back. :)

`Ideally I'd like an italic for each roman weight, in the above list.`

Sure. We will do this.

`Each of these should be its own family, in a Regular weight. Eg,`

I wanted to gather all of them in one family. I think what I understand is that it should be weight based rather than style based. So I'm gonna do your advice. :)

`And this is also true for KayKhosrow-Colored, but currently Google Fonts isn't supporting color fonts.
`

Maybe soon? :)

Great and thank you.

**Comment by shahabsiavash** [comment]
@davelab6 
Hi again :)

I updated the files. Please take a look and check if everything is ok: [KayKhosrow-Font](https://github.com/shahabsiavash/KayKhosrow-Font)

Thanks,

**Comment by shahabsiavash** [comment]
@davelab6 
The files and the readme are also updated. 

Please let me know if something else is necessary. :)

**Comment by shahabsiavash** [comment]
Is it August holidays in Google Fonts? :tropical_drink: :wink:

**Comment by davelab6** [comment]
hehe :) The final decision to include fonts in the Google Font collection isn't up to me individually; the collection is now curated by an internal Google team that I'm part of.  I'm focused on fixing bugs and improving some of the existing popular fonts, but I'll update this thread when I have news, probably in October or so :) Thanks for your patience with this!

**Comment by shahabsiavash** [comment]
Thanks. I know.
I wish they were here, too! :) We will wait until October then!
Good day,

**Comment by davelab6** [comment]
@shahabsiavash I'm sorry I never got back to you in this, it's great work but too experimental for us
